app/components/sanitize-html.js

```diff
@@ -8,7 +11,8 @@ export default class SanitizeHtmlComponent extends Component {
   }
 
   get sanitizedValue() {
-    const options = this.args.options;
+    const options = this.args.options || {};
+    options.allowedTags = isPresent(options.allowedTags) ? options.allowedTags : sanitizeHtml.defaults.allowedTags.concat(additionalAllowedTags); // add more tags to the default allowed tags
```

Comment:
Hier voorzie ik zelf nog de mogelijkheid om extra default tags toe te voegen (in ons geval `del`). Mocht dit qua structuur/opbouw iets anders mogen, laat gerust iets weten 👍